new/deprecations-in-resources.md

```diff
@@ -0,0 +1,89 @@
+---
+RFC: unassigned
+Title: Deprecation Warnings Within Custom Resources
+Author: Tim Smith <tsmith@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Deprecation Warnings Within Custom Resources
+
+If Chef 12 we introduced deprecation warnings within the chef-client. This allowed us to start communicating future breaking changes to users. The warnings and integration within Test Kitchen have become a powerful tool allowing users to future proof their cookbooks and ease chef-client migration projects.
```

Comment:
In Chef 12...

Reply:
I looks like it was 12.16 when we improved the way they worked based on our deprecations RFC

Reply:
Sorry, I meant to replace "If" with "In"